packages/hooks/src/use-boolean.ts

```diff
@@ -9,18 +9,11 @@ type InitialState = boolean | (() => boolean)
  */
 export function useBoolean(initialState: InitialState = false) {
   const [value, setValue] = useState(initialState)
+  const [switches] = useState(() => ({
+    on: () => setValue(true),
+    off: () => setValue(false),
+    toggle: () => setValue(prev => !prev),
+  }))
```

Comment:
question - what value does this provide over using useCallback

Reply:
Sorry, I've updated the hook to a slightly better version that uses `useMemo`.

Don't see much value in a double `useState`